src/modules/ads/ADS_SERVER_CONFIG_fbt.h

```diff
@@ -85,7 +85,6 @@ class FORTE_ADS_SERVER_CONFIG final : public CFunctionBlock {
     void setInitialValues() override;
 
   public:
-    FUNCTION_BLOCK_CTOR(FORTE_ADS_SERVER_CONFIG) {};
```

Comment:
what about the constructor of this one now?

Reply:
This is the same as the one below. This was anyhow an artifact left from some clean-up. Which means ADS was not compiling.